site/server/app/views/templates/library/exercise.scala.html

```diff
@@ -11,10 +11,11 @@
       </div>
 
       <div class="compile">
-        <button type="button" class="btn btn-default btn-xs" disabled>Compile</button>
```

Comment:
I set the button to `disabled` to prevent the user trying to evaluate unfinished exercises, do we want to have it enabled even when the exercises can not be compiled yet?